app/routines/newflow/update_salesforce_lead.rb

```diff
@@ -0,0 +1,59 @@
+module Newflow
+  class UpdateSalesforceLead
+    lev_routine
+
+    protected #################
+
+    def exec(user:)
+      status.set_job_name(self.class.name)
+      status.set_job_args(user: user.to_global_id.to_s)
+
+      lead_id = user.salesforce_lead_id
+      lead = fetch_lead(lead_id)
+
+      if not lead.present?
```

Comment:
it's standard to use the ! for not.  or even better, turn to an affirmative `if lead.blank?`

Reply:
replaced with `if lead.blank?` 👍